### PR TITLE
SIA R61: Ignore headings that are not in the accessibility tree

### PR DESCRIPTION
**Pull request by Jym77**

### CODE REVIEW

File: packages/alfa-rules/test/sia-r61/rule.spec.tsx [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
 import { test } from "@siteimprove/alfa-test";
 
-import { Document } from "@siteimprove/alfa-dom";
-
 import R61, { Outcomes } from "../../src/sia-r61/rule";
 
 import { evaluate } from "../common/evaluate";
 import { passed, failed, inapplicable } from "../common/outcome";
 
+const device = Device.standard();
```
Comment (kasperisager) [inline packages/alfa-rules/test/sia-r61/rule.spec.tsx:11]: There's no need to pass the standard device parameters to `evaluate()` and this is already the default if none are passed.

**Review by kasperisager — CHANGES_REQUESTED** [review]

File: packages/alfa-rules/test/sia-r61/rule.spec.tsx (hunk 1 above)
Comment (Jym77) [reply packages/alfa-rules/test/sia-r61/rule.spec.tsx:11]: Magic!
And I'm passing it so often 🙈

Comment (kasperisager) [reply packages/alfa-rules/test/sia-r61/rule.spec.tsx:11]: Object destructuring defaults are fantastic 😄 

https://github.com/Siteimprove/alfa/blob/7cfc5be90551302d4fc8beef34f4887f67d739de/packages/alfa-rules/test/common/evaluate.ts#L14-L19

**Review by kasperisager — APPROVED** [review]
💯

**Review by kasperisager — APPROVED** [review]

**Pull request merged**